cmake/common.cmake

```diff
@@ -0,0 +1,104 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
What's the purpose of this file in our case? Where and how it's used?

Reply:
It defines variables which are used by TRS and clientservice CMakeLists files.